embulk-input-mysql/src/test/java/org/embulk/input/mysql/BasicTest.java

```diff
@@ -41,6 +42,7 @@ private static String readResource(String fileName)
     @Before
     public void setup()
     {
+        TimeZone.setDefault(TimeZone.getTimeZone("Europe/Helsinki")); // +200
```

Comment:
`jvmArgs "-Duser.country=FI -Duser.timezone=Europe/Helsinki"` may not work in any environment.
But, in Linux `environment "TZ", "Europe/Helsinki"` may work and it doesn't work in Windows.